Parser/pegen.c

```diff
@@ -380,7 +377,7 @@ _PyPegen_raise_error(Parser *p, PyObject *errtype, const char *errmsg, ...)
 
 void *
 _PyPegen_raise_error_known_location(Parser *p, PyObject *errtype,
-                                    int lineno, int col_offset,
+                                    Py_ssize_t lineno, Py_ssize_t col_offset,
```

Comment:
Should we maybe change all of `t->lineno`, `t->col_offset` etc. to be of type `Py_ssize_t`?

Reply:
Yup!